packages/yourloops/test/app/main-lobby.test.tsx

```diff
@@ -53,6 +53,8 @@ describe("Main lobby", () => {
   const authPatient = loggedInUsers.patientSession;
   const authHookPatient: AuthContext = createAuthHookStubs(authPatient);
   const authHcpWithConsent = loggedInUsers.hcpSession;
+  const authNewHcp = loggedInUsers.newHcpSession;
```

Comment:
Are you forced to use this? Is it not possible to mock it? One of our goal is to get rid of this kind of stuff :slightly_smiling_face: